plugin/plugin.go

```diff
@@ -23,6 +26,25 @@ func New(policy string) validator.Plugin {
 
 func (p *plugin) Validate(ctx context.Context, req *validator.Request) error {
 
+	log.SetFormatter(&log.JSONFormatter{})
+	log.SetOutput(os.Stdout)
+
+	promotion := Promotion{
+		Build:       req.Build.Parent,
+		Environment: req.Build.Deploy,
+	}
+
+	// log each request for audit purposes
+	log.WithFields(log.Fields{
+		"extension":  "validation",
```

Comment:
nit: Adding a `type: audit` log field would make it easier to search for audit logs.

Reply:
good idea, adding it there.